### PR TITLE
Helm extra arg probably not needed?

### PR DESCRIPTION
**Pull request by ctr26**

### CODE REVIEW

File: charts/hypha/values.yaml [hunk 1]
```diff
@@ -59,25 +59,22 @@ redis:
   enabled: enabled
   global:
     redis:
-      password: "admin"
+      password: "redis-password"
```
Comment (oeway) [inline charts/hypha/values.yaml:62]: Is there security issue here if we set the password here? Maybe move to the values file?

File: charts/hypha/values.yaml [hunk 2]
```diff
@@ -59,25 +59,22 @@ redis:
...
 docker-registry:
   enabled: false
 
 minio:
-  # nameOverride: "minio"
   fullnameOverride: "minio"
   enabled: true
   rootUser: "minio"
-  rootPassword: "xZ/aDcP+PfoTWgwA9ZADTtU/+djwMeMKz4QlSKRcgEs="
+  rootPassword: "minio123"
```
Comment (oeway) [inline charts/hypha/values.yaml:71]: Set in the values file?

**Comment by ctr26** [comment]
You have to set a default value unfortunately On 5 Feb 2024, at 08:11, Wei Ouyang ***@***.***> wrote:﻿
@oeway commented on this pull request.



In charts/hypha/values.yaml:
> @@ -59,25 +59,22 @@ redis:
   enabled: enabled
   global:
     redis:
-      password: "admin"
+      password: "redis-password"

Is there security issue here if we set the password here? Maybe move to the values file?



In charts/hypha/values.yaml:
>    fullnameOverride: "minio"
   enabled: true
   rootUser: "minio"
-  rootPassword: "xZ/aDcP+PfoTWgwA9ZADTtU/+djwMeMKz4QlSKRcgEs="
+  rootPassword: "minio123"

Set in the values file?

**Comment by ctr26** [comment]
If you set these as blank then testing will fail

**Comment by ctr26** [comment]
I should define them as required fields though

**Pull request merged**